account_statement_import_txt_xlsx/models/account_statement_import_sheet_mapping.py

```diff
@@ -0,0 +1,234 @@
+# Copyright 2019 ForgeFlow, S.L.
+# Copyright 2020 CorporateHub (https://corporatehub.eu)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class AccountStatementImportSheetMapping(models.Model):
+    _name = "account.statement.import.sheet.mapping"
+    _description = "Bank Statement Import Sheet Mapping"
+
+    name = fields.Char(
+        required=True,
+    )
+    float_thousands_sep = fields.Selection(
+        string="Thousands Separator",
+        selection=[("dot", "dot (.)"), ("comma", "comma (,)"), ("none", "none")],
+        default="dot",
+    )
+    float_decimal_sep = fields.Selection(
+        string="Decimals Separator",
+        selection=[("dot", "dot (.)"), ("comma", "comma (,)"), ("none", "none")],
+        default="comma",
+    )
+    file_encoding = fields.Selection(
+        string="Encoding",
+        selection=[
+            ("utf-8", "UTF-8"),
+            ("utf-8-sig", "UTF-8 (with BOM)"),
+            ("utf-16", "UTF-16"),
+            ("utf-16-sig", "UTF-16 (with BOM)"),
+            ("windows-1252", "Western (Windows-1252)"),
+            ("iso-8859-1", "Western (Latin-1 / ISO 8859-1)"),
+            ("iso-8859-2", "Central European (Latin-2 / ISO 8859-2)"),
+            ("iso-8859-4", "Baltic (Latin-4 / ISO 8859-4)"),
+            ("big5", "Traditional Chinese (big5)"),
+            ("gb18030", "Unified Chinese (gb18030)"),
+            ("shift_jis", "Japanese (Shift JIS)"),
+            ("windows-1251", "Cyrillic (Windows-1251)"),
+            ("koi8_r", "Cyrillic (KOI8-R)"),
+            ("koi8_u", "Cyrillic (KOI8-U)"),
+        ],
+        default="utf-8",
+    )
+    delimiter = fields.Selection(
+        selection=[
+            ("dot", "dot (.)"),
+            ("comma", "comma (,)"),
+            ("semicolon", "semicolon (;)"),
+            ("tab", "tab"),
+            ("space", "space"),
+            ("n/a", "N/A"),
+        ],
+        default="comma",
+    )
+    quotechar = fields.Char(string="Text qualifier", size=1, default='"')
+    timestamp_format = fields.Char(required=True)
+    no_header = fields.Boolean(
+        string="File does not contain header line",
+        help="When this occurs please indicate the column number in the Columns section "
+        "instead of the column name, considering that the first column is 0",
+    )
+    timestamp_column = fields.Char(required=True)
+    currency_column = fields.Char(
+        help=(
+            "In case statement is multi-currency, column to get currency of "
+            "transaction from"
+        ),
+    )
+    amount_column = fields.Char(
+        help="Amount of transaction in journal's currency",
+    )
+
+    debit_column = fields.Char(
```

Comment:
This field (and `credit_column`) are defined 2 times, and I don't get why this has changed from `amount_debit_column` / `amount_credit_column`, and then they put an ugly related stored field. This duality should be undo and let it as it was in previous version.

Reply:
Done. In 15.0 there is only _amount_debit_column_ and `amount_credit_column` fields and no related fields.